src/WebRequest.cpp

```diff
@@ -310,13 +310,11 @@ bool AsyncWebServerRequest::_parseReqHeader(){
     if(name.equalsIgnoreCase("Host")){
       _host = value;
     } else if(name.equalsIgnoreCase("Content-Type")){
+	  _contentType = value.substring(0, value.indexOf(';'));
```

Comment:
what happens if there is no ';' in the value? A check should likely be made to assign the substring only if there is a ';' mark otherwise use the value as-is.

Reply:
If `value.indexOf` returns -1 (not found)[https://www.arduino.cc/reference/en/language/variables/data-types/string/functions/indexof/]

`value.substring` returns the complete string